File: 前端项目/ModelTest/Assets/MvRAssets/JS_Assets/JS_Scripts/Tile.js
//This script simply holds info about a tile. A tile is the object which buildings can be placed upon.
#pragma strict

private var thisTransform:Transform;

//The building which is currently placed on this tile
var building:Transform;

//The stackable which is currently placed on this tile
var stackable:Transform;

function Start() 
{
	thisTransform = transform;
}

File: 前端项目/ModelTest/Assets/MvRAssets/CS_Assets/CS_Scripts/Tile.cs
using UnityEngine;

namespace MvR
{
	/// <summary>
	/// This script simply holds info about a tile. A tile is the object which buildings can be placed upon.
	/// </summary>
	public class Tile : MonoBehaviour
	{
		// The building which is currently placed on this tile
		public Transform building;
		
		// The stackable which is currently placed on this tile
		public Transform stackable;
	}
}
